autoreject/autoreject.py

```diff
@@ -8,7 +8,7 @@
 from scipy.stats.distributions import uniform
 
 import mne
-from mne.utils import ProgressBar
+from tqdm import tqdm, trange
```

Comment:
lol I didn't expect to see you here @agramfort :) okay you got me thinking. Maybe I'll add a fallback to the `ProgressBar` from `mne.utils` ...